src/nanovg.c

```diff
@@ -2545,11 +2545,13 @@ int nvgTextBreakLines(NVGcontext* ctx, const char* string, const char* end, floa
 					breakMaxX = 0.0;
 				}
 			} else {
-				float nextWidth = iter.nextx - rowStartX;
```

Comment:
The comment does not make sense at it refers to some problem which is not described. Let's not focus on that yet.

So your use case was that you first call `nvgTextBoxBounds()` to effectively figure out how many lines there are and then draw using `nvgTextBox()` based on the returned bounds? If that is the case, it is exactly what I tried to solve with the current implementation.

That case is also really hard to get right because of the two effective bounds in fonts. For example big O or cursive small y can be optically aligned, so even if you ask to render them at certain location they actually start few pixels "before". In that case if you call text bounds and use the glyph bounds (geometry) to calculate stuff and render again (position in "logical" coordinates), you'll get offset results. It is complicated and annoying. That's the reason the rows have `width`(logical) as well as `minx`, `maxx` (geometry).

Can you post the code which you trigger the buggy case use? Let's see if there's some assumption there which fights against the two coordinate system stuff.